packages/bpk-component-theme-switcher/src/BpkThemeSwitcher.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { BpkButtonLink } from 'bpk-component-link';
+
+import { getHtmlElement, THEME_CHANGE_EVENT } from './utils';
+import bpkCustomTheme from './theming';
```

Comment:
This should also be configurable if we aim for this to be a public component

Reply:
That's the end goal, but walk before you run, we agreed on having an "hardcoded" theme for the moment and we'll decide what to do next